profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry_mock/response_templates/poetry-refuse-translation.tpl.php

```diff
@@ -40,7 +40,7 @@
   <?php foreach ($languages as $language): ?>
     <attributions format="<?php print $format ?>" lgCode="<?php print $language; ?>">
       <attributionsDelai><?php print date('d/m/Y H:i:s'); ?></attributionsDelai>
```

Comment:
No. 6 https://webgate.ec.europa.eu/fpfis/wikis/display/MULTISITE/Guide+into+code+reviewing+for+Drupal+7#GuideintocodereviewingforDrupal7-known_issuesQAknownissues